src/lib/components/versions/VersionForm.svelte

```diff
@@ -94,11 +94,13 @@
             <p>
               <strong>{$t('dependencies')}:</strong><br />
               {#each $modMeta.uplugin.Plugins as dependency}
-                <strong>{dependency.Name}: </strong>
-                {#if dependency.SemVersion}
-                  <span>{dependency.SemVersion}</span>
-                {:else}
-                  <span>*</span>
+                {#if dependency.Optional === false}
```

Comment:
Shouldn't it display an "optional" tag/text instead of hiding the dependency entirely? knowing what optional versions a mod cares about is still important